Source/Depressurizer/MainForm.cs

```diff
@@ -258,13 +258,18 @@ private static void CheckForDepressurizerUpdates()
 
 		private static ListViewItem CreateCategoryListViewItem(Category category)
 		{
-			return new ListViewItem(string.Format(CultureInfo.InvariantCulture, "{0} ({1})", category.Name, category.Count))
+			return new ListViewItem(CategoryListViewItemText(category))
 			{
 				Tag = category,
 				Name = category.Name
 			};
 		}
 
+		private static string CategoryListViewItemText(Category category)
```

Comment:
@alshain Could you make it so there are multiple overloads such as:
> CategoryListViewItemText (Category category) / (string categoryName, int categoryCount)

So this can work with [these ones](https://github.com/mvegter/Depressurizer/blob/dev/Source/Depressurizer/MainForm.cs#L1485).